neurobooth_os/tasks/saccade/saccade_task.py

```diff
@@ -105,6 +106,7 @@ def run_trials(self, prompt=True):
             self.target.pos = (0, 0)
             self.target.draw()
             self.win.flip()
+            self.update_tablet_background(0, 0)
             self.send_target_loc(self.target.pos)
```

Comment:
Should we bundle `send_target_loc` and `update_tablet_background` into one method in the base class so that target updates automatically trigger a background update?

Reply:
Kept these separate since they do fundamentally different things, even if mostly at the same time. Though in the saccade task there is a point where we want to update the tablet screen, but not sending out markers immediately, so they don't always occur together.